src/biogeochem/CNVegCarbonStateType.F90

```diff
@@ -216,6 +226,12 @@ subroutine InitAllocate(this, bounds)
     begg = bounds%begg; endg = bounds%endg
 
     allocate(this%leafc_patch              (begp:endp)) ; this%leafc_patch              (:) = nan
+!KO
```

Comment:
These variables don't seem to be used, maybe these can all be removed from the pull request?